### PR TITLE
DRY up code by introducing local macros

### PR DESCRIPTION
**Pull request by corny**

### CODE REVIEW

File: esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/ppd42ns-wificonfig-ppd-sds-dht.ino [hunk 1]
```diff
@@ -890,40 +905,46 @@ void webserver_config() {
 		page_content += F("</table><br/>");
 		page_content += F("<br/><input type='submit' name='submit' value='Speichern'/></form>");
 	} else {
-		if (server.hasArg("wlanssid") && server.arg("wlanssid") != "") {
-			server.arg("wlanssid").toCharArray(wlanssid,65);
-			if (server.hasArg("wlanpwd") && server.arg("wlanpwd") != "") {
-				server.arg("wlanpwd").toCharArray(wlanpwd,65);
```
Comment (corny) [inline esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/ppd42ns-wificonfig-ppd-sds-dht.ino]: A user wasn't able to remove the wifi password?

File: esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/ppd42ns-wificonfig-ppd-sds-dht.ino [hunk 2]
```diff
@@ -609,36 +616,41 @@ void readConfig() {
 				if (json.success()) {
 					debug_out(F("parsed json..."),DEBUG_MIN_INFO,1);
 					if (json.containsKey("SOFTWARE_VERSION")) strcpy(version_from_local_config, json["SOFTWARE_VERSION"]);
```
Comment (corny) [inline esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/ppd42ns-wificonfig-ppd-sds-dht.ino]: `version_from_local_config` isn't used anywhere. Can it be removed?

Comment (ricki-z) [inline esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/ppd42ns-wificonfig-ppd-sds-dht.ino]: No. This ist needed for future features.

File: esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/ppd42ns-wificonfig-ppd-sds-dht.ino (hunk 1 above)
Comment (ricki-z) [inline esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/ppd42ns-wificonfig-ppd-sds-dht.ino]: Check for empty ssid should be there. We had some problems with empty ssids. The check for an empty password could be removed.

**Comment by corny** [comment]
Why closed?

File: esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/ppd42ns-wificonfig-ppd-sds-dht.ino (hunk 2 above)
Comment (corny) [inline esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/ppd42ns-wificonfig-ppd-sds-dht.ino]: Then please add this explanation as a comment to the declaration of the variable.

**Comment by corny** [comment]
thanks

**Pull request merged**